### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by stramel**
Updates to similar layout as the lit eslint plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,97 +2,120 @@
...
+    "plugin:wc/recommended"
+  ]
 }
 ```
 
-Configure your rules like so:
+You should also specify settings that will be shared across all the plugin rules. ([More about eslint shared settings](https://eslint.org/docs/user-guide/configuring/configuration-files#adding-shared-settings))
 
-```json
+```jsonc
```
Comment (43081j) [inline README.md:41]: typo?

File: README.md [hunk 2]
```diff
@@ -2,97 +2,120 @@
...
-  "env": {"browser": true}
-}
-```
+<!-- TODO: Generate with `md-magic` -->
 
-### Custom settings
+### Possible Errors (Recommended)
 
-Settings are configured through a `settings` option in your eslintrc file.
+| Rule                                                                    | Description                                           | Fixable? |
```
Comment (43081j) [inline README.md:75]: can we turn this into a bullet list while we're here? so its the same as the lit plugin, lot easier to maintain

File: README.md (hunk 1 above)
Comment (stramel) [reply README.md:41]: Nah, jsonc is json with comments

File: README.md (hunk 2 above)
Comment (stramel) [reply README.md:75]: I was going to leave this here and add md-magic to generate the table automatically in a follow up pr

File: README.md (hunk 1 above)
Comment (43081j) [reply README.md:41]: TIL 😀

File: README.md (hunk 2 above)
Comment (43081j) [reply README.md:75]: tbh its never been filled in other than the description and you can infer that from the name usually. i'd keep it simple as a plain list personally

Comment (stramel) [reply README.md:75]: Okay, will push one later

**Review by 43081j — APPROVED** [review]

**Pull request merged**